src/Minitwit.API/Controllers/FollowController.cs

```diff
@@ -66,7 +65,7 @@ public async Task<IActionResult> ModifyFollow(string username, [FromBody] Follow
             return StatusCode(403, new {status = 403, error_msg = "You are not authorized to use this resource!"});
         }
 
-        if(!data.unfollow.IsNullOrEmpty())
+        if(!string.IsNullOrEmpty(data.unfollow))
```

Comment:
How and why is this so much better, i don't understand?

Reply:
same functionality but without the Microsoft.IdentityModel.Tokens import 🔫